app/models/patient.rb

```diff
@@ -134,10 +158,31 @@ def self.pledged_status_summary(num_days = 7)
     { pledged: outstanding_pledges, sent: sent_total }
   end
 
+  def self.to_csv
```

Comment:
This can go in the concern too (in a `class_methods` block, like in `app/models/concerns/callable.rb`)